Ansible/ansible_collections/jfrog/platform/roles/missionControl/files/searchguard/localhost.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Why are we checking in private ssh keys. This will likely be flagged by Git guardian.

Reply:
These are search gaurd related,  customers can override with new keys - same is applied for helm too